cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -2595,6 +2556,27 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 	});
 });
 
+describe("Hyperlink navigation check : Tests execution date and time : " + new Date(), () => {
+	it('Links through to "What did you think of this service? "', function () {
+		cy.visit(
```

Comment:
Looks good, but I think we should abstract this into a helper and do the same elsewhere - could make quite a few of the tests less verbose that way

Reply:
sure